app/register.js

```diff
@@ -33,20 +33,20 @@ export default function Login() {
 		<>
 			<StyledSafeArea
 				className="bg-offblack"
-				style={{ flex: 1 /*  backgroundColor: "#5946B2"  */ }}
+				style={{ flex: 1, backgroundColor: "#5946B2" }}
```

Comment:
like above, this should be converted to nativewind

Reply:
If you can, feel free to. @dev-arekusandoru